modules/perception/camera/lib/motion/plane_motion.cc

```diff
@@ -76,20 +73,16 @@ void PlaneMotion::generate_motion_matrix(base::VehicleStatus *vehicledata) {
     Eigen::AngleAxisf pitch_angle(pitch_delta, Eigen::Vector3f::UnitY());
     Eigen::AngleAxisf yaw_angle(yaw_delta, Eigen::Vector3f::UnitZ());
 
-    Eigen::Quaternion<float> q = roll_angle * pitch_angle * yaw_angle;
+    // here the intrinsic sequeeze is z-y-x(converted from z-x-y in location previously)
+    Eigen::Quaternion<float> q = yaw_angle * pitch_angle * roll_angle;
     Eigen::Matrix3f rot3d = q.matrix();
 
     float displacement = time_d * vehicledata->velocity;
     Eigen::Vector3f trans;
-    trans(0) =
-        static_cast<float>(sqrt(displacement * displacement /
-                                (tan(yaw_delta) * tan(yaw_delta) +
-                                 tan(pitch_delta) * tan(pitch_delta) + 1)));
-    trans(1) = static_cast<float>(tan(yaw_delta) * trans(0));
-    trans(2) = static_cast<float>(tan(pitch_delta) * trans(0));
+    trans << displacement, 0, 0;
 
     motion_3d.block(0, 0, 3, 3) = rot3d.transpose();
-    motion_3d.block(0, 3, 3, 1) = -rot3d.transpose() * trans;
+    motion_3d.block(0, 3, 3, 1) = -trans;
```

Comment:
Totally wrong change. motion_3D = [-RT | T]

Reply:
@techoe 
You mean motion_3D = [R^-1|-(R^-1)T], right?
And,
how about we calculate the trans as 
```
trans(0) = time_d * vehicledata->velocity_x;
trans(1) = time_d * vehicledata->velocity_y;
trans(2) = time_d * vehicledata->velocity_z;
```
I should be more accurate than the original one.
If so, I will create a new PR.